spec/features/single_content_item_spec.rb

```diff
@@ -65,20 +84,57 @@
 
     it 'renders the metric timeseries for unique_pageviews' do
       click_on 'Unique pageviews table'
-      unique_pageviews_rows = find("#unique_pageviews_2018-01-13-2018-01-15_table").all('tr')
-      pageviews_rows = find("#pageviews_2018-01-13-2018-01-15_table").all('tr')
+      unique_pageviews_rows = find("#unique_pageviews_2018-01-13-2018-01-15_table").all('tr').map do |el|
+        el.all('th,td').map(&:text)
+      end
+      expect(unique_pageviews_rows).to match_array([
+        ['', ''],
+        ['01-13', '101'],
+        ['01-14', '202'],
+        ['01-15', '303']
+      ])
+    end
 
-      expect(unique_pageviews_rows.count).to eq 4
-      expect(unique_pageviews_rows[0].text).to eq ''
-      expect(unique_pageviews_rows[1].text).to eq '01-13 101'
-      expect(unique_pageviews_rows[2].text).to eq '01-14 202'
-      expect(unique_pageviews_rows[3].text).to eq '01-15 303'
+    it 'renders the metric timeseries for pageviews' do
+      click_on 'Pageviews table'
+      pageviews_rows = find("#pageviews_2018-01-13-2018-01-15_table").all('tr').map do |el|
+        el.all('th,td').map(&:text)
+      end
 
-      expect(pageviews_rows.count).to eq 4
-      expect(pageviews_rows[0].text).to eq ''
-      expect(pageviews_rows[1].text).to eq '01-13 10'
-      expect(pageviews_rows[2].text).to eq '01-14 20'
-      expect(pageviews_rows[3].text).to eq '01-15 30'
+      expect(pageviews_rows).to match_array([
+        ['', ''],
+        ['01-13', '10'],
+        ['01-14', '20'],
+        ['01-15', '30']
+      ])
+    end
+
+    it 'renders the metric timeseries for on-page searches' do
+      click_on 'Number of internal searches table'
+      internal_searches_rows = find("#number_of_internal_searches_2018-01-13-2018-01-15_table").all('tr').map do |el|
```

Comment:
Can you extract this into a support method that returns the table content, this way we can reuse it anywhere.

As a side goal, can you please simplify the CSS selector? You might need to wrap the chart with a div, or add other classes to the table, but `number_of_internal_searches_2018-01-13-2018-01-15_table` feels like too complex for a test CSS selector.

Reply:
Done